### PR TITLE
Docs for Elements v7 

### PR DESCRIPTION
**Pull request by philsturgeon**

### CODE REVIEW

**Review by mmiask — COMMENTED** [review]
So far, it looks awesome! Very nicely structured and described in an easy, understandable manner. I left only a few comments, but nothing major :)

File: docs/getting-started/1-usage/web-component.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+</html>
+```
+
+For more configuration options, lets look at the properties available.
+
+## Configuration
+
+These properties can be provided as HTML attributes to the web component.
+
+- `apiDescriptionUrl` - OpenAPI document URL, supporting `http://`, `https://`, and documents containing `$ref` to other http(s) documents.
```
[the remark this answers is not in the document]
Comment (philsturgeon) [reply docs/getting-started/1-usage/web-component.md:58]: I believe we support relative files and external files, and if we dont we need to start working on that ASAP.

File: docs/getting-started/1-usage/web-component.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- `basePath` - Helps when using `router: 'history'` but docs are in a subdirectory like `https://example.com/docs/api`.
+- `layout` - There are two layouts for Elements:
+  - `sidebar` - (default) Three-column design.
+  - `stacked` - Everything in a single column, making integrations with existing websites that have their own sidebar or other columns already.
+- `router` -  Determines how navigation should work:
+  - `history` - (default) uses the HTML5 history API to keep the UI in sync with the URL.
+  - `hash` - uses the hash portion of the URL (i.e. window.location.hash) to keep the UI in sync with the URL.
+  - `memory` - keeps the history of your "URL" in memory (does not read or write to the address bar).
```
[the remark this answers is not in the document]
Comment (philsturgeon) [reply docs/getting-started/1-usage/web-component.md:66]: Aha! Will do.

File: docs/getting-started/1-usage/web-component.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+```html
+<elements-api
+  apiDescriptionUrl="https://api.apis.guru/v2/specs/github.com/1.1.4/openapi.yaml"
+  layout="stacked"
+/>
+```
+
+## Next steps
+
+Elements provides you with the components for beautiful API reference documentation, but you'll likely want to add other things to your site such as a landing page, header and footer navigation, etc. Check out our [Twitter Bootstrap](../../../examples/bootstrap/index.html) for an idea of how that could look.
```
[the remark this answers is not in the document]
Comment (philsturgeon) [reply docs/getting-started/1-usage/web-component.md:92]: Gonna worry about that another time. for now these are just artifacts for documentation, but #969 might change that.

**Comment by philsturgeon** [comment]
@mmiask can you quickly check what I wrote and hit merge if there is nothing glaringly problematic? I'd like to get this in as soon as we can and get more PRs with more content flowing as I get more written. This is better than whatever is there currently.

File: docs/getting-started/2-integrations/react.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- `apiDescriptionDocument` - OpenAPI document, provided as YAML string, JSON string or JavaScript object.
+- `basePath` - Helps when using `router: 'history'` but docs are in a subdirectory like `https://example.com/docs/api`.
+- `layout` - There are two layouts for Elements:
+  - `sidebar` - (default) Three-column design.
+  - `stacked` - Everything in a single column, making integrations with existing websites that have their own sidebar or other columns already.
+- `logo` - Object containing a few options for the small square logo next to the title, above the table of contents.
+  - `url` - URL of an image to use as the logo.
+  - `backgroundColor` - Set the background color with a hex code, e.g.: `"#0f0f0f`.
+  - `altText` - Set alternative text for screen-readers. 
+  - `href` - Link if somebody clicks the logo.
```
Comment (mmiask) [inline docs/getting-started/2-integrations/react.md:75]: Due to the limitations of web-components (passing string as `logo` value), the only accepted prop is `logo`

We have a full support (which you described) only for the API document with `x-logo` prop in `info` section. This is a limitation we agreed upon in #1202

File: docs/getting-started/1-usage/web-component.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Elements in HTML</title>
+  
+    <script src="https://unpkg.com/@stoplight/elements@beta/web-components.min.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements@beta/styles.min.css">
+  </head>
+  <body>
+
+    <elements-api
+      apiDescriptionUrl="tictactoe.yaml"
```
Comment (mmiask) [inline docs/getting-started/1-usage/web-component.md:44]: We should a prerequisite for downloading the `tictacotoe.yaml`, as in `/hello-world.md`

File: docs/guides/2-great-api-docs.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
+# Great API Documentation
```
Comment (mmiask) [inline docs/guides/2-great-api-docs.md:1]: This whole document is just ❤️

**Review by mmiask — COMMENTED** [review]
Let's change one thing and we are good to go! 💪

**Review by mmiask — APPROVED** [review]

**Pull request merged**